### PR TITLE
Fix for the issue of hearing audio and YouTube video when user is on feedback screen

### PR DESCRIPTION
**Pull request by MaximKhlobystov**
Issue #7198.

### CODE REVIEW

**Comment by antobinary** [comment]
Are webcams being viewed dropped when in Feedback?

File: bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
 import injectWbResizeEvent from '/imports/ui/components/presentation/resize-wrapper/component';
 import YouTube from 'react-youtube';
-import Vimeo from 'react-vimeo';
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx]: `    "react-vimeo": "^2.0.0",` would then be free to be removed from package.json, with package-lock.json consequences. @lfzawacki pls confirm this is no longer required

Comment (lfzawacki) [reply bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx]: Yes, you can remove it, this was never used.

**Comment by MaximKhlobystov** [comment]
We are going to explore doing a proper logout before showing the feedback screen, so I'm closing this PR.

**Pull request closed without merging**